app/views/shared/_provider_header_link.html.erb

```diff
@@ -1,6 +1,6 @@
 <ul id="navigation" class="govuk-header__navigation" aria-label="Top Level Navigation">
   <li class="govuk-header__navigation-item">
-    <%= link_to(current_provider.username.truncate(20), providers_provider_path, class: 'govuk-header__link') %>
+    <%= link_to(current_provider.username, providers_provider_path, class: 'govuk-header__link') %>
```

Comment:
I presume this is the "Ruby truncation" that you mentioned in your comments. So yes - removing the truncation here makes sense :+1:

Reply:
Yes.  This done in liaison with Julien.